tfvars.hcl.terraform-docs.yml

```diff
@@ -1,2 +1,12 @@
+# Original template for content: https://github.com/terraform-docs/terraform-docs/blob/v0.16.0/format/templates/tfvars_hcl.tmpl
+
+formatter: asciidoc
+content: |
+  {{ if .Module.Inputs -}}
```

Comment:
Can't we use the original template but with your fixes applied:
```diff
{{- if .Module.Inputs -}}
    {{- range $i, $k := .Module.Inputs -}}
      {{ if and $k.Description showDescription -}} 
          {{ convertToComment $k.Description }}
-          {{ align $k.Name $i }} = {{ value $k.GetValue }}
+         {{ align $k.Name $i }} = {{ $k.GetValue }}

      {{ else -}}
-          {{ align $k.Name $i }} = {{ value $k.GetValue }}
+         {{ align $k.Name $i }} = {{ $k.GetValue }}
      {{ end -}}
    {{ end -}}
{{- end -}}
```

Reply:
I've tried doing so, but got errors.
![image](https://github.com/dspace-group/simphera-reference-architecture-aws/assets/161831514/3dccb2d6-8222-4c77-a3e5-f48899809364)

Reply:
When if statement is removed there's another error:
![image](https://github.com/dspace-group/simphera-reference-architecture-aws/assets/161831514/781e3707-224a-46b5-830f-f0967979644c)

After removing function "convertToComment" there's error with the "align" function:
![image](https://github.com/dspace-group/simphera-reference-architecture-aws/assets/161831514/e6cb6709-80dc-4d9f-af72-a52dcf78b67c)

Therefore, the configuration that is introduced with this PR is a valid one